filament/src/main/java/net/fabricmc/filament/task/RemapUnpickDefinitionsTask.java

```diff
@@ -102,18 +102,27 @@ public void execute() {
 				final int toM = mappingTree.getNamespaceId(getParameters().getTargetNamespace().get());
 
 				for (MappingTree.ClassMapping classDef : mappingTree.getClasses()) {
-					classMappings.put(classDef.getName(fromM), classDef.getName(toM));
+					String fromClass = classDef.getName(fromM);
+					String toClass = classDef.getName(toM);
+
+					if (fromClass == null || fromClass.isEmpty()) {
+						fromClass = toClass;
+					} else if (toClass == null || toClass.isEmpty()) {
+						toClass = fromClass;
+					}
```

Comment:
Would the following work?
```suggestion
					if (fromClass == null || toClass == null) {
						contiune;
					}
```

Reply:
Doesn't the remap need the field remap tho ? The class and all its elements would be skipped in this case?

Reply:
In the case of those NPE, the from namespace is named and the to namespace is intermediary.